pom.xml

```diff
@@ -413,10 +419,10 @@
         <scope>runtime</scope>
       </dependency>
       <dependency>
-        <groupId>org.slf4j</groupId>
-        <artifactId>slf4j-simple</artifactId>
-        <version>${slf4j.version}</version>
-        <scope>runtime</scope>
+        <groupId>ch.qos.logback</groupId>
+        <artifactId>logback-classic</artifactId>
+        <version>${logback.version}</version>
+        <scope>test</scope>
```

Comment:
Should this have been left as `runtime` scope?

Reply:
> Should this be runtime scope, and if so shouldn't that be set above on line 327 rather than declaring the dependency twice?

Yes, agree. Now I understand how the `pom.xml` is structured. I made some bad assumptions when I updated the file. Doh! 🤦‍♂️

I can now see that the dependencies are ordered by scope. I moved the dependency to include it only once, along with the other runtime dependencies. 👍

Just waiting for local tests to pass before I commit.

Reply:
Done!